src/main/java/org/ays/common/port/adapter/AysAuditLogAdapter.java

```diff
@@ -0,0 +1,38 @@
+package org.ays.common.port.adapter;
+
+import lombok.RequiredArgsConstructor;
+import org.ays.common.model.AysAuditLog;
+import org.ays.common.model.entity.AysAuditLogEntity;
+import org.ays.common.model.mapper.AysAuditLogToEntityMapper;
+import org.ays.common.port.AysAuditLogSavePort;
+import org.ays.common.repository.AysAuditLogRepository;
+import org.springframework.stereotype.Component;
+
+/**
+ * Adapter for saving {@link AysAuditLog} entries into the audit log database.
+ * This class implements the {@link AysAuditLogSavePort} interface
+ * and utilizes the {@link AysAuditLogRepository} to persist audit log entities.
+ * It also uses a mapper to convert domain objects to entity objects.
+ */
+@Component
+@RequiredArgsConstructor
```

Comment:
@Transactional(readOnly = true) eklenilebilir sanki

Reply:
`@Transactional` anotasyonunu spring'in kendi repository metotlarını kullandığımızda yani JPA'den yararlandığımızda kullanıyoruz. Buradaki repository implementasyonunu biz yazdık ve kinesis client'ı biz çağırıyoruz. Hata olması durumunda transaction'ı kendisi otomatik yönetemeyecektir. Bu sebeple de ihtiyaç olmadığını düşünüyorum